### PR TITLE
Fix svg gulp task

### PR DESCRIPTION
**Pull request by Unetitebiere**

### CODE REVIEW

**Comment by wfortin** [comment]
:shipit:

File: package.json [hunk 1]
```diff
@@ -55,5 +55,8 @@
     "resources",
     "gulpfile.js",
     "LICENSE"
-  ]
+  ],
+  "dependencies": {
+    "gulp-files-to-json": "0.1.1"
```
Comment (JPLachance) [inline package.json]: I don't think this goes in dependencies. Plz move to devDependencies.

**Comment by JPLachance** [comment]
Look's good, ship it when your PR in the Admin will have been approved.

File: gulpTasks/svg.js [hunk 2]
```diff
@@ -1,38 +1,37 @@
...
+                removeComments: true    
+            }]
         }))
-        .pipe(rename({prefix: 'coveo-icon-'}))
-        .pipe(svgstore({
-            inlineSvg: true
+        .pipe(cheerio(function ($, file) {
+            // Each file will be run through cheerio and each corresponding `$` will be passed here.
+            // `file` is the gulp file object
+            // Make all h1 tags uppercase
```
Comment (wfortin) [inline gulpTasks/svg.js]: is this comment really needed ?

File: gulpTasks/svg.js [hunk 3]
```diff
@@ -1,38 +1,37 @@
...
+            }, {
+                removeComments: true    
+            }]
         }))
-        .pipe(rename({prefix: 'coveo-icon-'}))
-        .pipe(svgstore({
-            inlineSvg: true
+        .pipe(cheerio(function ($, file) {
+            // Each file will be run through cheerio and each corresponding `$` will be passed here.
+            // `file` is the gulp file object
```
Comment (wfortin) [inline gulpTasks/svg.js]: I dont see any usage of `file`, you could just remove it from the arguments.

**Comment by wfortin** [comment]
LGTM

**Comment by marilynechan** [comment]
LGTM, Sketch seems to be adding a lot of cra.. *stuff in the SVGs.

**Pull request merged**